replica/table.cc

```diff
@@ -2550,6 +2550,14 @@ table::disable_auto_compaction() {
     });
 }
 
+void table::set_tombstone_gc_enabled(bool tombstone_gc_enabled) noexcept {
+    _tombstone_gc_enabled = tombstone_gc_enabled;
+    tlogger.info0("Tombstone GC was {} for {}.{}", tombstone_gc_enabled ? "enabled" : "disabled", _schema->ks_name(), _schema->cf_name());
+    if (_tombstone_gc_enabled) {
+        trigger_compaction();
+    }
```

Comment:
This setting is not persisted, so as soon as the API call returns, the setting may be reverted by a node restart. Is that ok? 

I think we should document the purpose of this API to illustrate what are the expectations.

Reply:
> This setting is not persisted, so as soon as the API call returns, the setting may be reverted by a node restart. Is that ok?
> 
> I think we should document the purpose of this API to illustrate what are the expectations.

Yes. The idea is to provide persistence with the help of a yaml file, will come next.